### ISSUE TITLE
 Error: _fs2.default.existsSync(...)

### ISSUE THREAD

**Issue by wolfgangvonludwigsburg** [ModuleLoader/browser-es-module-loader#3]
On Win7, fresh install of node.js (node-v6.9.1-x64.msi),

taking your demo/Usage:

    <script src="dist/babel-browser-build.js"></script>
    <script src="dist/browser-es-module-loader.js"></script>

    
    <script type="module">
        import {x} from './y.js';
    </script>

I receive the error:

    browser-es-module-loader.js:74 Uncaught (in promise) Error: _fs2.default.existsSync is not a function(…)
    LoaderError__Check_error_message_for_loader_stack @ browser-es-module-loader.js:74
    (anonymous function) @ browser-es-module-loader.js:142

thrown by:

    babel-browser-build.js:
    1685:    return existsCache[filename] = _fs2.default.existsSync(filename);

.existsSync seems to be deprecated ...

Another concern would then be:
- is the ES6 spread-operator and rest-parameter part of this package?
- how would I configure 'stage-...' settings

Please don't beat me if I'am wrong, since I just recently started into node.js ...

I would be glad, you'd give me some workaround ...

**Comment by guybedford** [comment]
I believe this is because `existsSync` was deprecated in NodeJS but as far as I'm aware it was then added back again. Ensure you're running the latest version of NodeJS perhaps?

This demo project doesn't include any other ES features, it just uses Babel for modules support.

**Comment by wolfgangvonludwigsburg** [comment]
Dear Guy,

many thanks for your response!

Sure, installed versions are node v6.9.1, npm 3.10.8
And you are right, fs.existsSync() is still available
in this actual **node context** version.

I only can report based on my (bootstrapping...) knowlege(?).

I found my older sort of functioning test-case
where in babel-browser-build.js, older Version

    1700: return existsCache[filename] = _pathExists2.default.sync(filename);

this line now changed in actual Version to

    1685: return existsCache[filename] = _fs2.default.existsSync(filename);

But as we are in the **browser context**,
fs defaults to a "raw Object"
i.e. only default base Object props,
i.e. no fs.existsSync() is available

I found kind and part of a browserify based fs emulating solution (npm brfs),
but could'nt investigate just further

For me your 'browser-es-module-loader.js'
seems far more comfortable (at least for prototyping),
rather than dealing with webpack based bundling developments ...

Maybe I'm totally wrong,
best wishes, Wolfgang

PS: Are'nt spread/rest already committed parts of ES6 ?
